ui/src/app/views/dataset/component.tsx

```diff
@@ -31,17 +40,42 @@ export const styles = {
   }
 };
 
+const ResourceListItem: React.FunctionComponent <
+  {conceptSet: ConceptSet, openConfirmDelete: Function, edit: Function}
+  > = ({conceptSet, openConfirmDelete, edit}) => {
+    return<div style={{border: '0.5px solid #C3C3C3', margin: '.4rem',
+      height: '1.5rem', display: 'flex'}}>
+      <div style={{width: '.75rem', paddingTop: 5, paddingLeft: 10}}>
+        <ResourceCardMenu disabled={false}
+                          resourceType={ResourceType.CONCEPT_SET}
+                          onDeleteResource={openConfirmDelete}
+                          onEditConceptSet={edit}/>
+      </div>
+      <input type='checkbox' value={conceptSet.name} style={{height: 17, width: 17,
```

Comment:
I think it would probably valuable for this ticket to also cover managing selected concept sets. I don't think we can do anything with them yet, but it would probably be good to work out that behavior sooner rather than later.

Reply:
Ok, will definitely add that in then, wasn't sure.